Zad1/src/sql/demo/Main.java

```diff
@@ -0,0 +1,60 @@
+package sql.demo;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.Scanner;
+
+public class Main {
+
+
+    public static void main(String[] args) throws Exception{
```

Comment:
[ERROR] /Users/geoiva/Desktop/SberJava/students/alena_kochurova/nordic-it-java-q4/Zad1/src/sql/demo/Main.java:10:60: '{' is not preceded with whitespace. [WhitespaceAround]
[ERROR] /Users/geoiva/Desktop/SberJava/students/alena_kochurova/nordic-it-java-q4/Zad1/src/sql/demo/Main.java:10:60: WhitespaceAround: '{' is not preceded with whitespace. [WhitespaceAround]